RELEASE_PROCESS.md

```diff
@@ -3,86 +3,201 @@
+is testable by users and signalled in the form of
+`<major>.<minor>.<patch>-rc.<rc-number>`.
+
+If a release candidate is created, the `-dev`-suffix is removed and the suffix
+is appended to generate the name of the release.
+
+During the release, just before creating the git tag for a release,
+the [VERSION](./VERSION) file is changed to include this new suffix.
+The transformation thus looks like
+
+```shell
+<major>.<minor>.<patch>-dev -> <major>.<minor>.<patch>-rc.<rc-number>
+```
+
+### Creating a Minor release
+
+Once a release candidate is seen as sufficiently tested, the release manager can
+promote the release candidate to a full release.
+
+By default one should always create a draft release first (as a Release
+Candidate),
+open it up for testing (by communicating the new release candidate as available
+to stakeholders),
+and after a grace period, promote the draft release to a full release.
+
+This promotion is currently effectively a full rebuild from the release branch,
+with the difference that the `-rc.<rc-number>` suffix is removed.
+
+After the build, instead of finishing, the [
+`release`](./.github/workflows/release.yaml) GitHub Action will also publish the
+release.
+
+This publishing to package registries (such as brew) is delegated to [
+`publish-to-other-than-github`](./.github/workflows/publish-to-other-than-github.yaml)
+
+After the release on a release branch was successful, because the release was
+now run without a suffix, the version is considered `burned`.
+This means that, even if bugs are found for that release in the future, a
+`patch` release will be created for that release branch.
+
+Concretely this means that the following additional steps are taken:
+
+1. The release is tagged with the version number from the [`VERSION`](./VERSION)
+   file, *without* the `-rc.<rc-number>` suffix.
+2. The release is published on GitHub as the latest release, not as a
+   pre-release.
+3. The release is published to the package registries.
+4. The release branch is bumped in [`VERSION`](./VERSION) to the next <patch>,
+   e.g. `0.17.0-dev` becomes `0.17.1-dev` via Pull Request.
+
+### Creating a Patch release
+
+The process to creating a Patch release is almost equivalent to the process of
+creating a Minor release.
+
+Whenever a patch release in the form of <major>.<minor>.<patch> is needed, the
+branch `releases/<major>.<minor>` is used to prepare the release.
+The only difference is that now the VERSION file should contain the suffix
+`<major>.<minor>.<patch>-dev`
+(which should have automatically been bumped on the last release).
+
+This means that creating candidates for the patch or creating the release is
+equivalent by using the [`release`](./.github/workflows/release.yaml) GitHub
+Action.
+Make sure that all patched commits have been cherry-picked from main.
+
+*NOTE: It is not valid to create a fresh commit on a patch branch without a
+corresponding cherry-pick from main. Use the next minor version for features.*
```

Comment:
> Use the next minor version for features.

I don't get that... context is patch release process... which doesn't allow features...

Reply:
youre right its obvious, i removed that part